source/docs/prismic/index.md

```diff
@@ -0,0 +1,10 @@
+---
+id: prismic-overview
+title: Overview
+---
+
+The Front-Commerce Prismic module brings support for the headless CMS Prismic to Front-Commerce. For now, it allows to replace Magento2's CMS pages and blocks system with content coming from a Prismic repository.
+
+- [Setup](/docs/prismic/setup.html)
```

Comment:
We use `Installation` for most of the existing pages so far. It might be good to remain consistent here:
```suggestion
- [Installation](/docs/prismic/installation.html)
```

Note: the page will have to be renamed too, along with the config.

## Existing "Installation" pages

- https://developers.front-commerce.com/docs/essentials/installation.html
- https://developers.front-commerce.com/docs/magento1/installation.html
- https://developers.front-commerce.com/docs/advanced/shipping/mondial-relay.html#Installation
- https://developers.front-commerce.com/docs/advanced/payments/adyen.html#Install-and-configure-the-Adyen-Payment-Magento2-extension-6-5 (Install vs Setup)

Reply:
fixed in 21e1cac